README.md

```diff
@@ -1,43 +1,36 @@
 # GOV.UK Display Screen
 
-Screen to display live data from GOV.UK. Includes number of people on GOV.UK,
-latest searches, trending and recent content. Not publicly accessible because
-there's sometimes personal data in the latest searches.
+Screen to display live data from GOV.UK. It displays the number of GOV.UK users in last 30 mins, live searches, 
+most popular pages and recent content. It is intended as an internal tool only because the live searches are unfiltered.
```

Comment:
Maybe add in a bit about the graph: It displays the number of GOV.UK users in the last 30 mins, _a user count graph from the past day_, live searches, most popular pages and recent content.